feature-find-meal/src/main/java/com/kinandcarta/feature/find/meal/viewmodel/DisplayMealsViewModel.kt

```diff
@@ -11,21 +12,23 @@ import com.google.android.gms.location.FusedLocationProviderClient
 import com.google.android.gms.location.LocationCallback
 import com.google.android.gms.location.LocationRequest
 import com.google.android.gms.location.LocationResult
+import com.google.android.gms.maps.model.LatLng
 import com.kcc.kmmhackathon.shared.MealsSDK
 import com.kcc.kmmhackathon.shared.entity.Meal
 import com.kcc.kmmhackathon.shared.utility.DistanceUnit
 import kotlinx.coroutines.launch
 
-typealias Meals = MutableList<Meal>
+typealias Meals = List<Meal>
 
-class FindMealViewModel @ViewModelInject constructor(
-    private val fusedLocationClient: FusedLocationProviderClient
+class DisplayMealsViewModel @ViewModelInject constructor(
```

Comment:
If you'd like this ViewModel to be unit testable (run it as a standalone unit test, not instrumentation test on a device) it shouldn't reference anything from the Android system.

This would mean that interactions with the Operating System should not happen here. Maybe you need to start / stop listening for locations from your view, and delegate new locations received to the ViewModel, which will handle all of your business logic and expose livedata for your view to then observe and react to.